src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -105,6 +105,10 @@ template <size_t Dim, typename Frame, typename DataType>
 struct TraceSpacetimeChristoffelFirstKind : db::SimpleTag {
   using type = tnsr::a<DataType, Dim, Frame>;
 };
+template <size_t Dim, typename Frame, typename DataType>
```

Comment:
I'm not sure why only some of the tags in this file are documented, but could you add something brief just so this shows up in doxygen?